scripts/travis/package-deploy.sh

```diff
@@ -67,7 +63,8 @@ rm -rf deploy $DEPLOY_STAGING_DIR
 mkdir deploy
 mkdir $DEPLOY_STAGING_DIR
 
-package linux
+package linux-amd64
 package darwin
```

Comment:
Q: what would the directory name be for MacOS/Windows? At the moment all platforms have uniform file names, e.g. jaeger-1.18.0-windows-amd64.tar.gz

Reply:
Ahhh ok then I will add the architecture for MacOS and Windows. Thanks for pointing this out :)